state/enableha.go

```diff
@@ -116,7 +116,8 @@ func (st *State) EnableHA(
 			return nil, errors.Trace(err)
 		}
 		if desiredControllerCount == 0 {
-			desiredControllerCount = votingCount
+			// Make sure we go to add odd number of desired voters. Even if HA was currently at 2 desired voters
+			desiredControllerCount = votingCount + (votingCount+1)%1
```

Comment:
Should this be modulo 2?

Reply:
I think you're right, but for some reason the tests do pass, and they should be exercising this code.